docs/pages/web/banneroverlay.js

```diff
@@ -0,0 +1,150 @@
+// @flow strict
+import { Fragment, type Node as ReactNode } from 'react';
+import { BannerOverlay, FixedZIndex, Image, Text } from 'gestalt';
+import AccessibilitySection from '../../docs-components/AccessibilitySection';
+import docGen, { type DocGen } from '../../docs-components/docgen';
+import GeneratedPropTable from '../../docs-components/GeneratedPropTable';
+import InternalDocumentationSection from '../../docs-components/InternalDocumentationSection';
+import MainSection from '../../docs-components/MainSection';
+import Page from '../../docs-components/Page';
+import PageHeader from '../../docs-components/PageHeader';
+import QualityChecklist from '../../docs-components/QualityChecklist';
+import SandpackExample from '../../docs-components/SandpackExample';
+import main from '../../examples/banneroverlay/main';
+
+export default function DocsPage({ generatedDocGen }: { generatedDocGen: DocGen }): ReactNode {
+  return (
+    <Fragment>
+      <BannerOverlay
```

Comment:
can we just leave one banneroverlay? haha it gets confusing with the main example one or place them at tdifferent heights

Reply:
Yeah, this is just there while documentation work is still pending. I'll leave this unresolved to remind myself to remove it before final approval.

Reply:
DONE